### ISSUE TITLE
Failure while translating Erlang's logger event 

### ISSUE THREAD

**Issue by danbaranov** [elixir-lang/elixir#9162]
Hey there! Getting this issue after bumping elixir and OTP versions.

```

Failure while translating Erlang's logger event 
** (FunctionClauseError) no function clause matching in Logger.Translator.sup_name/1
     (logger) lib/logger/translator.ex:318: Logger.Translator.sup_name(:refetch_consumer_supervisor)
     (logger) lib/logger/translator.ex:270: Logger.Translator.report_supervisor/2
     (logger) lib/logger/erlang_handler.ex:104: Logger.ErlangHandler.translate/6
     (logger) lib/logger/erlang_handler.ex:97: Logger.ErlangHandler.translate/5
     (logger) lib/logger/erlang_handler.ex:27: anonymous fn/3 in Logger.ErlangHandler.log/2     (logger) lib/logger.ex:861: Logger.normalize_message/2
     (logger) lib/logger.ex:684: Logger.__do_log__/3
     (kernel) logger_backend.erl:51: :logger_backend.call_handlers/3
```

OTP 22
Elixir 1.7.4

According to https://github.com/elixir-lang/elixir/issues/8126 it supposed to be resolved. Any ideas where to look at?

**Comment by josevalim** [comment]
@danbaranov that's a different error. Do you have a mechanism for us to reproduce it?

**Comment by danbaranov** [comment]
It's hard to make a synthetic suite to reproduce but I might try later this week. From what I see now it feels to me that's it's somehow related to a situation when there's no more data to be pulled from GenStage Producer by Consumer Supervisor.

Btw I upgraded to Elixir 1.8.4 and nothing has changed.

**Comment by josevalim** [comment]
That makes sense. ConsumerSupervisor is actually the culprit here because it is passing the wrong format but I think we should add a catch all anyway.

**Comment by josevalim** [comment]
Closed by 1586801de94fb889e1f66d1d3e5b3fdb8cae01cc. I accidentally linked to the other issue. :)
